Lib/test/test_pty.py

```diff
@@ -230,6 +231,10 @@ def test_fork(self):
                 os._exit(2)
             os._exit(4)
         else:
+            flags = fcntl.fcntl(master_fd, fcntl.F_GETFD)
+            cloexec_set = bool(flags & fcntl.FD_CLOEXEC)
+            flags = fcntl.fcntl(master_fd, fcntl.F_GETFD)
+            self.assertTrue(flags & fcntl.FD_CLOEXEC)
```

Comment:
I'm sorry, I got it wrong, and now there are duplicates, please apply these changes and everything should be fine.
```suggestion
            flags = fcntl.fcntl(master_fd, fcntl.F_GETFD)
            self.assertTrue(flags & fcntl.FD_CLOEXEC)
```